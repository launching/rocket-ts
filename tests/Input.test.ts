import Vue from 'vue';
import ElementUI from 'element-ui';

import { shallowMount, mount } from '@vue/test-utils';
import RInput from '@/components/widget/Input.vue';

Vue.use(ElementUI);

describe('RInput.vue', () => {
  it('render form', async () => {
    const wrapper = shallowMount(RInput, {
      propsData: {
        name: 'name',
      },
      provide: {
        model: { name: '' },
        changeModelItem: () => {},
        initModelItemValidate: () => {},
      },
    });
    await wrapper.vm.$nextTick();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
