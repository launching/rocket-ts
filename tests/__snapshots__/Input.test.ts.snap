// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RInput.vue render form 1`] = `
<div class="r-input">
  <div class="el-input">
    <!----><input type="string" autocomplete="off" class="el-input__inner">
    <!---->
    <!---->
    <!---->
    <!---->
  </div>
  <div class="r-toolbar"></div>
</div>
`;
